### ISSUE TITLE
We are running BatchNorm2D with batch_size = 1

### ISSUE THREAD

**Issue by manuelli** [RobotLocomotion/pytorch-dense-correspondence#82]
It turns out this isn't a big issue because we are doing BatchNorm2D. Straight from the pytorch documentation [here](http://pytorch.org/docs/master/nn.html?highlight=batchnorm2d#torch.nn.BatchNorm2d)

> Because the Batch Normalization is done over the C dimension, computing statistics on (N, H, W) slices, it’s common terminology to call this Spatial Batch Normalization

So even though we have `N = 1`, the fact that `H,W` are greater than `1` makes the batch normalization work fine.
